server/game/cards/03-WC/Memolith.js

```diff
@@ -0,0 +1,39 @@
+const Card = require('../../Card.js');
+
+class Memolith extends Card {
+    setupCardAbilities(ability) {
+        this.action({
+            targets: {
+                graftOrTrigger: {
+                    mode: 'select',
+                    choices: {
+                        'Graft an action card': (context) => context.player.hand.filter(card => card.type === 'action'),
+                        'Trigger play effect': (context) => context.source.childCards.some(card => !card.facedown)
```

Comment:
Same here with the facedown check, I don't like that "faceup === grafted", it needs to be more explicit